src/components/Authentication/SignUpUser.tsx

```diff
@@ -205,7 +205,7 @@ const SignUpUser = () => {
 
 													<Field id='Signupemail'
 														name='email'
-														type="email"
+														// type="email"
```

Comment:
please remove the commented code if not required

Reply:
resolved